settings.json.template

```diff
@@ -409,6 +409,25 @@
   "indentationOnNewLine": false,
   */
 
+  /*
+   * Import Rate limiting, File size and If a session is required to import
+   * 
+   * importRateLimiting defaults at 10 requests per IP per 90 seconds.  After this it's rejected, value is in mS.  See express-rate-limit for more options
+   * requireAuthorSessionToImport defaults to true meaning that an author must be on the pad to import.
+   * importMaxFileSize defaults to 50Mb meaning that a single imported file max size is 50Mb.  File size is specified in bytes.
+   * 
+   * Uncomment to customize.
```

Comment:
I think it's better leaving this section uncommented: after all, now import/export is always going to be rate limited.

New installas will inherit this file, and have the config knob ready. Old installs will use the default mechanism you implemented in Settings.js.

Reply:
Sounds good to me!  Happy for that change to be made :)  Sorry if I'm being defective in making these easy to make behavioral changes btw, I'm super deep into the whole list issues and it's reaching the limit of my mental capacity.

Reply:
Ok agreed.

The new settings for import/export checks on size and author existence + rate limiting **will be already uncommented** in `config.json.template`. **Their values will be equal to the default ones**.

If old installs won't upgrade the configuration file, will inherit the default settings via Settings.js.

This is the backwards-compatibility mechanism that is already in place, btw.